apps/site/lib/site_web/controllers/schedule/line_controller.ex

```diff
@@ -128,6 +128,7 @@ defmodule SiteWeb.ScheduleController.LineController do
     |> services_by_route_id_fn.()
     |> dedup_identical_services()
     |> dedup_similar_services()
+    |> Enum.reject(fn service -> service.id == "canonical" end)
```

Comment:
question (non-blocking): Is there a reason for filtering by ID rather than the [typicality](https://mbta.slack.com/archives/C03K6NLKKD1/p1684263685617409?thread_ts=1684169905.222479&cid=C03K6NLKKD1)? Supporting API documentation of canonical typicality can be found [here](https://github.com/mbta/api/blob/35d4f329c73f8ee1a0131c54e1759dda5420f851/apps/api_web/lib/api_web/controllers/service_controller.ex#L163)

Reply:
Yeah I am moving it over to typicality instead of id.  I wasn't aware it was a new typicality.  dotcom set it to unknown.  Adding the new typicality shortly.